src/quo2/screens/drawers/action_drawers.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.screens.drawers.action-drawers
+  (:require [quo.previews.preview :as preview]
+            [quo.react-native :as rn]
+            [quo2.components.drawers.action-drawers :as quo2]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]))
+
+(def descriptor [{:label   "Muted?"
+                  :key     :muted?
+                  :type    :boolean}
+                 {:label   "Show red options?"
+                  :key     :show-red-options?
+                  :type    :boolean}])
+
+(def options-with-consequences [{:icon :main-icons2/share-context
+                                 :label "Clear history"}])
+
+(defn cool-preview []
+  (let [state         (reagent/atom {:muted?         true
+                                     :show-red-options? true})]
+    (fn []
+      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+       [rn/view {:padding-bottom 400}
+        [preview/customizer state descriptor]
+        [rn/view {:padding-vertical 60}
+         [quo2/action-drawer {:actions-with-consequence (when (:show-red-options? @state) options-with-consequences)
```

Comment:
would be also great to have a button to open drawer in the bottom sheet

Reply:
yeah I think it's a good suggestion 👍